src/Seq.App.EmailPlus/SmtpOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MailKit.Security;
+
+namespace Seq.App.EmailPlus
+{
+    class SmtpOptions
+    {
+        public string Server { get; set; } = string.Empty;
+        public int Port { get; set; } = 25;
+        public string User { get; set; } = string.Empty;
+        public string Password { get; set; } = string.Empty;
+        public bool UseSsl { get; set; } = false;
+        public bool RequiresAuthentication { get; set; } = false;
+        public string PreferredEncoding { get; set; } = string.Empty;
+        public SecureSocketOptions? SocketOptions { get; set; }
```

Comment:
`SocketOptions` and `UseSsl` overlap, here. Perhaps we knock out `UseSsl`, make `SocketOptions` non-nullable, and set it to `SslOnConnect` when using SSL, or `StartTlsWhenAvailable` otherwise? (This is how MailKit interprets Boolean `useSsl` behind the scenes.

Reply:
@nblumhardt  I think you're right. I'll take a look.

Reply:
SocketOptions is now the only way to fly